base/logger_external.go

```diff
@@ -73,6 +74,29 @@ func (GoCBCoreLogger) Log(level gocbcore.LogLevel, offset int, format string, v
 	return nil
 }
 
+// ******************************************************
+// SG Replicate Logging
+// ******************************************************
+// Log levels are mapped as follows:
+//   Debug   -> SG Debug
+//   Normal  -> SG Info
+//   Warning -> SG Warn
+//   Error   -> SG Error
+//   Panic   -> no-op
+
+var sgreplicateLogFn = func(level clog.LogLevel, format string, args ...interface{}) {
```

Comment:
Like on the sg-replicate PR, it would be better if this was an actual function instead of a variable to a function - the variable is never going to change.